README.md

```diff
@@ -65,16 +65,6 @@ You only need to do this if you are working on mentoring-related endpoints.
 * Head back to [Google Developer Console](https://console.developers.google.com/apis/library). Go to the same project you created before. Go to Libraries and search for Google Calendar API. Enable that API.
 * Next go to Credentials Tab. Click Create Credentials > API Key. You can store this key in `keys/google.json` under `web.api_key` or set the ENV variable `GOOGLE_API_KEY`.
 
-### Google Calendar
```

Comment:
Why remove this? IIRC, the section above is for OAuth and this one is for setting up Google Calendar.

Reply:
See above, there's a duplicate of the gcal stuff right above it